docs/src/pages/components/cdk-components/date-input/examples/basicDateInput.js

```diff
@@ -0,0 +1,40 @@
+import { DxcDateInput } from "@dxc-technology/halstack-react";
+import { useState } from "react";
+
+const code = `() => {
+  const options = [
+    { label: "Option 01", value: "1" },
+    { label: "Option 02", value: "2" },
+    { label: "Option 03", value: "3" },
+    { label: "Option 04", value: "4" },
+  ];
+
+  return (
+    <div style={{ display: "flex" }}>
```

Comment:
Maybe is a good time to start using our layout components in these examples.

Reply:
We can take this into account when we change the Example component, what do you think?

Reply:
Since we are going to change the way the examples are presented we can do this in the future.